spyderlib/plugins/editor.py

```diff
@@ -626,6 +626,14 @@ def get_plugin_actions(self):
         add_shortcut_to_tooltip(self.new_action, context="Editor",
                                 name="New file")
         
+        self.open_last_closed = create_action(self, _("O&pen last closed"),
+                icon=ima.icon('filenew'), tip=_("Open last closed"),
+                triggered=self.open_last_closed)
+        self.register_shortcut(self.open_last_closed, context="Editor",
+                               name="Open last closed")
+        add_shortcut_to_tooltip(self.open_last_closed, context="Editor",
+                                name="Open last closed")

```

Comment:
Please remove the this `add_shortcut_to_tooltip` addition. It's not needed in this case.